packages/atomic/src/components/commerce/product-template-components/atomic-product-text/atomic-product-text.tsx

```diff
@@ -51,6 +82,11 @@ export class AtomicProductText
     }
   }
 
+  private get fieldForHighlights() {
+    const cleanedField = this.field.replace(/^ec_/, '');
```

Comment:
nit: we _could_ make this function simpler by explicitly making it handle the `ec_name` and `excerpt` fields.

Otherwise, the function gives the impression it can highlight other fields.